src/cli/commands/dev.ts

```diff
@@ -73,6 +74,14 @@ export class Dev implements Command {
       dev: {
         addToWatcherSettings: {},
         async onBeforeWatcherStartOrRestart(change) {
+          if (change.type === 'change') {
+            const nexusModules = Layout.findNexusModules(
+              layout.tsConfig,
```

Comment:
OT: We don't currently support changes to tsConfig in dev mode correct? We should create an issue about that.

Reply:
True! I'll do it now!